MVP/README.md

```diff
@@ -0,0 +1,76 @@
+# Platform-CI MVP
+
+## What's MVP?
+
+Minimal Viable Product (MVP) is a minimized set of Jenkins job definition (and supporting files) on Platform CI:
+
+  - for End-to-End workflow changes (batch update) via **Template / Default YAML**
+  - for create/update/scale tests (by individuals) via **Test / Job YAML**
+
+## How to create a CI job with MVP?
+
+1. ####Get MVP files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/MVP
+
+1. ####Tweak [sample_job.yaml](/MVP/sample_job.yaml/) with your own test parameters
+
+    *Job YAMLs store parameters which are specific for your testing jobs. It's recommended to maintain them in your team git repo.*
+
```

Comment:
Basically yes, I don't think there is a simple way to maintain both sides in one repo - so it has to be maintained seperately.

Plan A: Instead of copying files around everytime, I was thinking about sym-links from platform-ci template files to their repo (without `git add` them) folder.
Plan B: https://platform-stg-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Platform-CI-MVP-Job-Builder/build